tests/tests_config.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Test cockpit_* role options
+  hosts: all
+  tasks:
+    - import_role:
+        name:
+          linux-system-roles.cockpit
+      vars:
+        cockpit_packages: minimal
+        cockpit_enabled: yes
+        cockpit_started: no
+        cockpit_config:
+          WebService:
+            LoginTitle: "hello world"
+          Session:
+            IdleTimeout: 60
+
+    - meta: flush_handlers
+
+    - name: test - socket is enabled  # noqa 303 command-instead-of-module
+      command: "systemctl is-enabled {{ __cockpit_daemon }}"
+      changed_when: false
+
+    - name: test - socket is not active
+      shell: |
+        ! systemctl is-active {{ __cockpit_daemon }}
+      changed_when: false
+
+    - name: test - cockpit-bridge is installed
+      stat:
+        path: /usr/bin/cockpit-bridge
+      register: result
+      failed_when: not result.stat.exists
+
+    - name: test - cockpit metapackage is not installed
+      stat:
+        path: /usr/share/man/man1/cockpit.1.gz
+      register: result
+      failed_when: result.stat.exists
+
+    - name: test - expected configuration file
+      copy:
+        content: |+
+          #
+          # Ansible managed
+          #
+          [WebService]
+          LoginTitle = hello world
+
+          [Session]
+          IdleTimeout = 60
+
+        dest: /etc/cockpit/cockpit.conf
+      check_mode: yes
+      register: result
+      failed_when: result.changed
+
+    - import_role:
+        name: tests-cleanup
```

Comment:
Please don't use a role for this - just implement the cleanup tasks using a file like `tests/tasks/cleanup.yml` then use `include_tasks: tasks/cleanup.yml` in your test playbooks - using a role for this unnecessarily complicates things